### PR TITLE
Format `block-meta.json` with prettier

### PR DESCRIPTION
**Pull request by dkarnutsch**
- [x] Add changeset (if necessary)

### CODE REVIEW

File: packages/api/cms-api/src/blocks/blocks-meta.service.ts [hunk 1]
```diff
@@ -1,9 +1,15 @@
 import { getBlocksMeta } from "@comet/blocks-api";
 import { OnModuleInit } from "@nestjs/common";
 import { promises as fs } from "fs";
+import { format, resolveConfig } from "prettier";
```
Comment (johnnyomair) [inline packages/api/cms-api/src/blocks/blocks-meta.service.ts:4]: Prettier is a dev dependency, so this wouldn't work in production, right? Wouldn't it be a better solution to do this in a precommit hook with lint-staged?

Comment (johnnyomair) [reply packages/api/cms-api/src/blocks/blocks-meta.service.ts:4]: Also, block-meta.json in `@comet/cms-api` isn't formatted

Comment (johnnyomair) [reply packages/api/cms-api/src/blocks/blocks-meta.service.ts:4]: It think it's even valid to ignore the block-meta.json file for Prettier

Comment (dkarnutsch) [reply packages/api/cms-api/src/blocks/blocks-meta.service.ts:4]: Agree.

**Pull request closed without merging**